### PR TITLE
CEDS-2357 - Migrate 'Total Package Quantity' page to Gov Design System

### PR DESCRIPTION
**Pull request by iservport**

### CODE REVIEW

**Comment by PauloPortugal** [comment]
Please ensure the Smoke tests are passing.

File: app/views/components/gds/Styles.scala [hunk 1]
```diff
@@ -21,4 +21,5 @@ object Styles {
   val gdsPageLegend = "govuk-fieldset__legend--l"
   val gdsPageHeading = "govuk-heading-l"
   val gdsPageLabel = "govuk-label--l"
+  val gdsPageLabelXl = "govuk-label--xl"
```
Comment (SteveSugden) [inline app/views/components/gds/Styles.scala:24]: You don't need this.  Once all pages are GDS we can change gdsPageLabel to "xl"

File: app/views/declaration/total_package_quantity.scala.html [hunk 2]
```diff
@@ -17,40 +17,51 @@
...
+        @sectionHeader(messages("supplementary.items"))
 
-        @components.fields.field_text(
+        @exportsInputText(
+            labelKey = "declaration.totalPackageQuantity.title",
             field = form("totalPackage"),
-            label = Some(""),
-            labelClass = Some("")
+            isPageHeading = true,
+            headingClasses = gdsPageLabelXl,
```
Comment (SteveSugden) [inline app/views/declaration/total_package_quantity.scala.html:59]: Use gdsPageLabel

**Comment by iservport** [comment]
> Please ensure the Smoke tests are passing.
Smoke tests are passing.

**Pull request merged**